### PR TITLE
Feature/filter by costs test

### PR DESCRIPTION
**Pull request by Denzel-Turambi**
What I did:
- Created recipesFromPrice to filter recipes by the cost of the recipe
- Created recipesFromPrice describe and it blocks to test the functions
- Refactored HTML to include an input box and label for users budget
- Add style rules for the new input box

Did this break anything?
- [x] No

Type of change
- [x] New feature (non-breaking change which adds functionality)

Checklist:
- [x]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [x]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next?
DOM Updated utilizing the recipeFromPrice function

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -47,6 +47,12 @@ <h1 class="title" tabindex="0">What's Cookin?</h1>
   </main>
   <div class="all-recipes-wrapper">
     <aside class="all-filters hidden">
+      <div class="filter-option">
+        <form class="input-budget">
+          <input type="text" placeholder="$$$" class="budget-input-box" id="budget-input" name="budget-input" required>
```
Comment (andreasorensen) [inline dist/index.html:52]: Yesss!

File: src/recipeUtils.js [hunk 2]
```diff
@@ -51,6 +51,13 @@ const shuffleData = (recipes) => {
   return
 }
 
+const recipesFromPrice = (recipes, maxCost, ingredients) => {
+  return recipes.filter((recipe) => {
+    const cost = calculateRecipeCost(recipe, ingredients);
```
Comment (andreasorensen) [inline src/recipeUtils.js:56]: Clean!

**Review by andreasorensen — APPROVED** [review]
This all looks really good!

**Review by fernandorobles97 — APPROVED** [review]

**Review by hvohr — APPROVED** [review]
Love the form and required usage

**Pull request merged**